js/modules/k6/execution/execution_test.go

```diff
@@ -111,10 +111,12 @@ func TestVUTags(t *testing.T) {
 				v   interface{}
 				exp string
 			}{
-				"string": {v: `"tag1"`, exp: "tag1"},
-				"bool":   {v: true, exp: "true"},
-				"int":    {v: 101, exp: "101"},
-				"float":  {v: 3.14, exp: "3.14"},
+				"undefined": {v: `null`, exp: ""},
+				"null":      {v: `undefined`, exp: ""},
```

Comment:
```suggestion
				"null":           {v: `null`, exp: ""},
				"undefined": {v: `undefined`, exp: ""},
```
Not that we use the map key anywhere ... but it seems to be the "name" of the test in all other cases ;)